### ISSUE TITLE
I cant create a token grabber

### ISSUE THREAD

**Issue by LittleMC** [Rdimo/Hazard-Nuker#237]
*Please provide the following info:*

**Description:*Hi i cant create a token grabber*

What's expected to happen: create a .exe file

What actually happens: crashes the app

**How to recreate this bug:* I select 13 and i name the executable but its crashing the app*

**(Optional but preferred) Screenshots/Video:**

**Comment by MJMODZZ** [comment]
> _Please provide the following info:_
> 
> **Description:_Hi i cant create a token grabber_
> 
> What's expected to happen: create a .exe file
> 
> What actually happens: crashes the app
> 
> __How to recreate this bug:_ I select 13 and i name the executable but its crashing the app_
> 
> **(Optional but preferred) Screenshots/Video:**

Its mostly caused by a wrong installed pyinstaller. If you need help with that, dm me on discord MJMODZZ#6094

**Comment by GodFather227** [comment]
> > _Please provide the following info:_
> > **Description:_Hi i cant create a token grabber_
> > What's expected to happen: create a .exe file
> > What actually happens: crashes the app
> > __How to recreate this bug:_ I select 13 and i name the executable but its crashing the app_
> > **(Optional but preferred) Screenshots/Video:**
> 
> Its mostly caused by a wrong installed pyinstaller. If you need help with that, dm me on discord MJMODZZ#609

no, option 13 plain and simple doesnt work very well. Use hazard grabber v2 instead

**Comment by MJMODZZ** [comment]
> > > _Please provide the following info:_
> > > **Description:_Hi i cant create a token grabber_
> > > What's expected to happen: create a .exe file
> > > What actually happens: crashes the app
> > > __How to recreate this bug:_ I select 13 and i name the executable but its crashing the app_
> > > **(Optional but preferred) Screenshots/Video:**
> > 
> > 
> > Its mostly caused by a wrong installed pyinstaller. If you need help with that, dm me on discord MJMODZZ#609
> 
> no, option 13 plain and simple doesnt work very well. Use hazard grabber v2 instead

It works for me. Like i said its caused by a wrong installed pyinstaller. The reason to dont use Hazard-Token-Grabber is because its not Encrypter like Option 13, Option 13 has lower detections.

**Comment by GodFather227** [comment]
> > > > _Please provide the following info:_
> > > > **Description:_Hi i cant create a token grabber_
> > > > What's expected to happen: create a .exe file
> > > > What actually happens: crashes the app
> > > > __How to recreate this bug:_ I select 13 and i name the executable but its crashing the app_
> > > > **(Optional but preferred) Screenshots/Video:**
> > > 
> > > 
> > > Its mostly caused by a wrong installed pyinstaller. If you need help with that, dm me on discord MJMODZZ#609
> > 
> > 
> > no, option 13 plain and simple doesnt work very well. Use hazard grabber v2 instead
> 
> It works for me. Like i said its caused by a wrong installed pyinstaller. The reason to dont use Hazard-Token-Grabber is because its not Encrypter like Option 13, Option 13 has lower detections.

im talking mainly about the exe not src

**Comment by MJMODZZ** [comment]
> > > > > _Please provide the following info:_
> > > > > **Description:_Hi i cant create a token grabber_
> > > > > What's expected to happen: create a .exe file
> > > > > What actually happens: crashes the app
> > > > > __How to recreate this bug:_ I select 13 and i name the executable but its crashing the app_
> > > > > **(Optional but preferred) Screenshots/Video:**
> > > > 
> > > > 
> > > > Its mostly caused by a wrong installed pyinstaller. If you need help with that, dm me on discord MJMODZZ#609
> > > 
> > > 
> > > no, option 13 plain and simple doesnt work very well. Use hazard grabber v2 instead
> > 
> > 
> > It works for me. Like i said its caused by a wrong installed pyinstaller. The reason to dont use Hazard-Token-Grabber is because its not Encrypter like Option 13, Option 13 has lower detections.
> 
> im talking mainly about the exe not src

you are right. I recomment to use https://github.com/Smug246/Luna-Token-Grabber , its the best TG you can get righ now using discord injection from rdimo

**Comment by GodFather227** [comment]
> > > > > > _Please provide the following info:_
> > > > > > **Description:_Hi i cant create a token grabber_
> > > > > > What's expected to happen: create a .exe file
> > > > > > What actually happens: crashes the app
> > > > > > __How to recreate this bug:_ I select 13 and i name the executable but its crashing the app_
> > > > > > **(Optional but preferred) Screenshots/Video:**
> > > > > 
> > > > > 
> > > > > Its mostly caused by a wrong installed pyinstaller. If you need help with that, dm me on discord MJMODZZ#609
> > > > 
> > > > 
> > > > no, option 13 plain and simple doesnt work very well. Use hazard grabber v2 instead
> > > 
> > > 
> > > It works for me. Like i said its caused by a wrong installed pyinstaller. The reason to dont use Hazard-Token-Grabber is because its not Encrypter like Option 13, Option 13 has lower detections.
> > 
> > 
> > im talking mainly about the exe not src
> 
> you are right. I recomment to use https://github.com/Smug246/Luna-Token-Grabber , its the best TG you can get righ now using discord injection from rdimo

fuck no smug can lick my dick

**Comment by GodFather227** [comment]
they all are basically the same lmao

**Comment by dekrypted** [comment]
> > > > > > _Please provide the following info:_
> > > > > > **Description:_Hi i cant create a token grabber_
> > > > > > What's expected to happen: create a .exe file
> > > > > > What actually happens: crashes the app
> > > > > > __How to recreate this bug:_ I select 13 and i name the executable but its crashing the app_
> > > > > > **(Optional but preferred) Screenshots/Video:**
> > > > > 
> > > > > 
> > > > > Its mostly caused by a wrong installed pyinstaller. If you need help with that, dm me on discord MJMODZZ#609
> > > > 
> > > > 
> > > > no, option 13 plain and simple doesnt work very well. Use hazard grabber v2 instead
> > > 
> > > 
> > > It works for me. Like i said its caused by a wrong installed pyinstaller. The reason to dont use Hazard-Token-Grabber is because its not Encrypter like Option 13, Option 13 has lower detections.
> > 
> > 
> > im talking mainly about the exe not src
> 
> you are right. I recomment to use https://github.com/Smug246/Luna-Token-Grabber , its the best TG you can get righ now using discord injection from rdimo

luna is buggy, and the dev had me help with half the shit 💀 if you want hazard to be undetected use a obfuscator
